support-frontend/stories/screens/supporterPlusThankYou/AusSupporterPlusThankYou.stories.tsx

```diff
@@ -17,7 +17,6 @@ import {
 	setEmail,
 	setFirstName,
 	setLastName,
-	setUserTypeFromIdentityResponse,
```

Comment:
We should delete these stories wholesale since it's a page no longer used, but work for another PR